docs/modules/help/pages/users_edit.adoc

```diff
@@ -0,0 +1,25 @@
+= Adding or modifying users
+
+WARNING: Depending on your access rights, your actual screen might differ from what is shown in the screenshots in this help system.
+
+[[overview]]
+== Overview
+
+Users with the "Customer Admin" role can create and edit users. Clicking on the btn:[Create User] button prompts the user creation form.
+
+When clicking on an individual user entry, the same form appears pre-filled with the data of that user.
+
+.User creation/edition form
+image::users-edit.png[]
+
+<1> Use this section to enter basic data for the user.
+<2> Select a role for the user in this section.
```

Comment:
There's really a *lot* more to be said about this.
* Which role does what? (manager = do everything except edit users, admin = do everything including edit users, accounting = see invoices & read/write tickets, appuio = use appuio, developer = see everything except invoices, create tickets, manage openshift projects, ticket = read/write tickets)
* Can I/should I select multiple roles? (yes you can, but normally you don't need to)